ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/cmake/tasks/main.yml

```diff
@@ -45,8 +39,9 @@
     - (ansible_architecture == "x86_64") or (ansible_architecture == "ppc64le")
   tags: cmake
 
+# NOTE: PATH setting is for RHEL6/CentOS6 since gcc 4.4.7 not C++11 compliant
```

Comment:
"is not"

Reply:
Oooh that's a real nitpick :-) Sorted both issues